conda_forge_tick/update_recipe/version.py

```diff
@@ -773,3 +897,30 @@ def update_version(raw_meta_yaml, version, hash_type="sha256"):
     else:
         logger.critical("Recipe did not change in version migration!")
         return None, errors
+
```

Comment:
Let's remove this debugging code.

Reply:
Why was this resolved? We need to remove this code. The little entry points like this rot inplace.

Reply:
I removed it now. I only removed similar code in another file last time - a mistake!